cli/src/semdep/external/parsy/README.rst

```diff
@@ -0,0 +1,48 @@
+parsy
```

Comment:
Can you add a section somewhere (maybe a diff Readme.md file) on why we are vendoring this and pointing to the specific changes you made